doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1202,7 +1202,7 @@ The move tactic.
          Goal not False.
          move.
 
-   More precisely, the :tacn:`move` tactic inspects the goal and does nothing
+   More precisely, the :tacn:`move (ssreflect)` tactic inspects the goal and does nothing
```

Comment:
What is the relation of this change with the changes to the conversion chapter? I'm OK for this change but please keep it in a separate commit, make sure that all references to this tactic in this chapter are changed, and maybe use `<move (ssreflect)>` to avoid having the parenthesis shown in the text.

Reply:
I used <move (ssreflect)>.  Making it a separate commit (or worse, a separate PR) seems like overkill.  It's akin to fixing a misspelling when you notice it.  Make typos too tedious to fix and they will remain unfixed.  You think?

Reply:
The fact that you find it tedious to move it to a separate commit means that you have new git skills to learn. I would agree that it would be overkill and would discourage typo fixing if it was actually difficult. And it would also be a bad thing to make this a requirement for occasional contributors. But since you are a major contributor, I find it worth to tell you that we usually prefer commits that are well separated semantic units (I'm not talking about separate PRs for trivialities here).

In this case, separating the PR in three commits is particularly easy since the changes to separate are in separate files. This is just about doing the right `git add` followed by `git commit`. But even when the changes are in the same file (but in distinct places), it can be relatively easy to do (see https://nuclearsquid.com/writings/git-add/).

Since it's slightly more difficult to do after the fact that during the preparation of the commits, I'll do it for you in this PR.

Reply:
I've put the main commit last so that it is easier to edit / squash for you later. Remember to pull the new version of the PR.

>I used <move (ssreflect)>.

You didn't do this change, so I did it for you.

Reply:
I definitely made that change but somehow lost it.

`git add --patch` is itself tedious, particularly if there are many changes to include or exclude.  And a command-diff is often not very usable.  There were a few other link labels such as `apply (ssreflect)` that I meant to fix.  Grouping those now with the other in the second commit now requires a rebase.  I think these are trivialities.  But we don't have to agree 100%.

Reply:
But squashing at this time was not helpful.  I need to see what I changed in the last round of changes to make sense of your comments.  I refer to the review commits constantly.

Reply:
Ah sorry about that. I thought the review comments were meant for the reviewers only. I didn't realize they were useful for you as well.